scapy/tools/UTscapy.py

```diff
@@ -490,6 +491,12 @@ def run_campaign(test_campaign, get_interactive_session, verb=3, ignore_globals=
                     passed += 1
                 else:
                     failed += 1
+                    if drop_to_interpreter:
+                        code.interact(banner="Test '%s' failed. "
+                                             "exit() to stop, Ctrl-D to leave "
```

Comment:
Just out of curiosity, I recognized that `exit()` and `Ctrl-D` act not identical on this interpreter. `exit()` kills the whole process (`UTScapy`) and `Ctrl-D` just exits from the interactive interpreter and still continues the campaign. Do you know why that is the case?

Reply:
I have no idea =/